TOC/README.md

```diff
@@ -0,0 +1,88 @@
+|  ----  | ----  |  ----  | ----  |
+| Xiaoguang Sun sunxiaoguang@pingcap.com  | [sunxiaoguang](https://github.com/sunxiaoguang) | PingCAP | TOC Chairman |
+| Siddon Tang tl@pingcap.com | [siddontang](https://github.com/siddontang) | PingCAP | TOC Official Member |
+| Chengxuan Xu bigpyer@gmail.com | [niubell](https://github.com/niubell) | PalFish | TOC Official Member |
+| Yuqing Bai baiyuqing@zhihu.com | [baiyuqing](https://github.com/baiyuqing) | Zhihu | TOC Official Member |
+| Jian Zhang zhangjian@pingcap.com | [zz-jason](https://github.com/zz-jason) | TiDB | TOC Official Member |
+| Jinpeng Zhang zhangjinpeng@pingcap.com | [zhangjinpeng1987](https://github.com/zhangjinpeng1987) | TiKV | TOC Official Member |
+| Chengwen Yin cwen@pingcap.com | [cwen0](https://github.com/cwen0) | Chaos Mesh®  | TOC Official Member |
+| Dian Luo luodian@pingcap.com | [andylokandy](https://github.com/andylokandy) | TiKV | TOC Non-voting Member |
+| Jay Li jay@pingcap.com | [BusyJay](https://github.com/BusyJay) | TiKV | TOC Non-voting Member |
+| Cong Wang wangcong@pingcap.com | [bb7133](https://github.com/bb7133) | TiDB | TOC Non-voting Member |
+
+
+### Emeritus Members
+
+TOC members has no concept of tenure, but will retire under the following circumstances:
+
+- Actively choose to retire due to personal reasons.
+- When he/she can no longer participate in TOC affairs on behalf of this organization due to leaving or withdrawing from the organization they represent.
+- The organization he/she represents no longer exists.
+
+Retirees will become emeritus members of TOC. The list of emeritus members is as below.
+
+- Xianlin Chen，Head of Technology Hub at PalFish
+- Qiang Zhou，Chaos Mesh® Maintainer
+
+
+## TOC Meeting
+The TOC monthly online meeting is held at 8 pm on the last Friday of each month. Meeting invitations will be sent through the TOC mail group: https://lists.tidb.io/g/toc. 
+
+If official members cannot attend the monthly meeting for special reasons, they must ask for leave in advance.
+
+### Meeting Minutes
+
+[English version](https://docs.google.com/document/d/1m9Uk-JkqO_KaSZaLjL47hW1L_2Bp-yW1StCuGQzOFTk/edit)
+
+[Chinese version](https://docs.google.com/document/d/1KvavvwWpABbTEJO3IMLBykGMOUMq4soL2ityDryDqgw/edit)
+
+## Decision Making
+Matters that require TOC members to make decisions can be proposed at the TOC meeting or indicated by a pull request adding an entry under the [Proposal folder](./Proposal).
+
+All TOC members have the right to make proposals. Only official members have the right to vote, and each has 1 vote. Decisions are made by lazy consensus. If anyone votes against it, discussion is needed. If no agreement can be reached, Majority Vote will be used.
+
+
+| Actions | Description | Approval | Binding Voters |  Minimum Length (days) |
+|:----|:----|:----|:----|:----|
+| New TOC Member |  When a new TOC member is nominated.  |  [Lazy Consensus](./Proposal/voting-rules.md#lazy-consensus)  |  TOC Official Members  |  6  |
```

Comment:
You said 2 days in `voting-rules.md`

> For all votes, voting must be open for at least 2 days.

Reply:
at least 2 days.

Reply:
here you wrote Minimum Length 6 days. These two information confict.

Reply:
The rule "at least 2 days" is for all votes. As long as the voting time is more than 2 days, it is in line with the rules. 
Here the vote "New TOC Member" must last for at least 6 days. They don't conflict.

Reply:
Make sense. Thanks for your clarification.